scripts/clean.mjs

```diff
@@ -0,0 +1,79 @@
+import FastGlob from "fast-glob";
+import { existsSync, lstatSync, readdirSync, rmdirSync, unlinkSync } from "fs";
+import { join } from "path";
+
+/**
+ * This script is used to clean up build artifacts.
+ * - Removes all node_modules folders if `--reset`-argv is passed
+ */
+const globPatterns = [
+  "./@navikt/**/lib",
+  "./@navikt/**/esm",
+  "./@navikt/**/cjs",
+  "./@navikt/aksel-icons/src",
+  "**/playwright-report",
+  "**/.next",
+  "**/dist",
```

Comment:
I think we should have `"**/dist"` first. Now it first deletes `@navikt/aksel-icons/dist/react/cjs` and then `@navikt/aksel-icons/dist`...